packages/web/src/components/common/pool-graph/PoolGraph.tsx

```diff
@@ -417,23 +417,45 @@ const PoolGraph: React.FC<PoolGraphProps> = ({
       tokenA: tokenA,
       tokenB: tokenB,
       tokenAAmount: tokenAAmountStr
-        ? convertToKMB(tokenAAmountStr.toString())
+        ? formatTokenExchangeRate(tokenAAmountStr.toString(), {
+          maxSignificantDigits: 6,
+          minLimit: 0.000001,
+          fixedDecimalDigits: 6
+        })
         : "-",
       tokenBAmount: tokenBAmountStr
-        ? convertToKMB(tokenBAmountStr.toString())
+        ? formatTokenExchangeRate(tokenBAmountStr.toString(), {
+          maxSignificantDigits: 6,
+          minLimit: 0.000001,
+          fixedDecimalDigits: 6
+        })
         : "-",
       myTokenAAmount:
         index < 20
           ? "-"
           : index > 19 && `${currentBin.reserveTokenAMyAmount}` === "0"
             ? "<0.000001"
-            : convertToKMB((myTokenAAmountStr || "-").toString()) || "-",
+            : formatTokenExchangeRate((myTokenAAmountStr || "-").toString(), {
+              maxSignificantDigits: 6,
+              minLimit: 0.000001,
+              fixedDecimalDigits: 6
+            }) || "-",
+      // myTokenAAmount:
+      //   index < 20
+      //     ? "-"
+      //     : index > 19 && `${currentBin.reserveTokenAMyAmount}` === "0"
+      //       ? "<0.000001"
+      //       : convertToKMB((myTokenAAmountStr || "-").toString()) || "-",
       myTokenBAmount:
         index > 19
           ? "-"
           : index < 20 && `${currentBin.reserveTokenBMyAmount}` === "0"
             ? "<0.000001"
-            : convertToKMB((myTokenBAmountStr || "-").toString()) || "-",
+            : formatTokenExchangeRate((myTokenBAmountStr || "-").toString(), {
```

Comment:
Is the logic correct to put a `"-"` string when there is falsy value for `myTokenBAmountStr` ?
```
formatTokenExchangeRate((myTokenBAmountStr || "-").toString(), {
```

Reply:
It will not wrong fỏ this case cause any NaN value will be transformed to `"-"`, however i can remove it now